invokeai/frontend/web/src/features/canvas/hooks/useColorUnderCursor.ts

```diff
@@ -33,6 +33,10 @@ const useColorPicker = () => {
           1
         ).data;
 
+      if (!(a && r && g && b)) {
```

Comment:
Even when it makes sense for variables to be 1 letter long, it still so PTSD inducing from my university math days lol

Reply:
I promise there aren't many of these! This is only in the PR due to typescript refactoring, else you'd never have known